### PR TITLE
[studio] chore: clear-all button placement update

### PR DESCRIPTION
**Pull request by grzim**
Zrzut ekranu 2024-07-11 o 19 47 13

### CODE REVIEW

File: apps/frontend/app/(website)/studio/features/modGPT/PromptPanel/PromptPanel.tsx [hunk 1]
```diff
@@ -7,8 +8,10 @@ import type { useAiService } from "@chatbot/useAiService/useAiService";
 import type { useModGptSubmit } from "@chatbot/useAiService/useModGpt/useModGptSubmit";
 import { getOrderedAliasList, usePrompts } from "@chatbot/utils";
 import { useAuth } from "@clerk/nextjs";
+import ButtonWithTooltip from "@studio/components/button/BottonWithTooltip";
 import { useGetAliases } from "@studio/store/CFS/alias";
 import type { UseChatHelpers } from "ai/react";
+import Link from "next/link";
 import { useRef, useState } from "react";
```
Comment (r4zendev) [inline apps/frontend/app/(website)/studio/features/modGPT/PromptPanel/PromptPanel.tsx:15]: unused imports

File: apps/frontend/app/(website)/studio/features/modGPT/PromptPanel/ClearHistoryButton.tsx [hunk 2]
```diff
@@ -0,0 +1,19 @@
+import { ArrowElbowDownLeft } from "@phosphor-icons/react";
+import { Trash as TrashIcon } from "@phosphor-icons/react/dist/csr/Trash";
```
Comment (r4zendev) [inline apps/frontend/app/(website)/studio/features/modGPT/PromptPanel/ClearHistoryButton.tsx:2]: if you have some spare time, let's make sure we use icons from one single package and remove the other one.
shadcn that we are using in studio uses lucide icons by default, some other places do use it too. phosphor icons were brought to us by Tinloof i believe, if we like them more, let's replace all the other usages.

**Review by r4zendev — APPROVED** [review]
approving, but please remove unused imports.

**Comment by grzim** [comment]
@r4zendev, I totally forgot about this PR! This was meant to be closed. Sorry I made you waste your time on this:(

**Pull request closed without merging**